builtin/credential/okta/backend.go

```diff
@@ -246,7 +268,7 @@ func (b *backend) Login(ctx context.Context, req *logical.Request, username, pas
 				}
 
 				select {
-				case <-time.After(500 * time.Millisecond):
+				case <-time.After(1 * time.Second):
```

Comment:
Taking the opportunity in this PR to change the polling frequency from 500ms to 1s. The rationale behind this change is that Okta dev accounts has a rate limit of 100req/min on `authn/` endpoints, and at this frequency calls made by a vault login will trigger a warning and eventually lead to rate limit exceeded errors. By lowering this to a max of 60req/min avoid hitting this rate limit and getting into flaky testing scenarios. The bump should not be that noticeable since this is a purely human-based flow.

Login MFA uses a 1s polling frequency, so it should be acceptable for us here as well: https://github.com/hashicorp/vault/blob/a2fc885b03d1121e64796632c5cc63320cc5d510/vault/login_mfa.go#L1928